Outline and update the Data and Code Availability section in Methods (#20)
d3b-center/OpenPedCan-methods: content/06.methods.md

[efo-mondo-mapping](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/efo-mondo-mapping)
[filter-mtp-tables](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/filter-mtp-tables)
[focal-cn-file-preparation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/focal-cn-file-preparation)
[fusion-frequencies](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/fusion-frequencies)
[fusion-summary](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/fusion-summary)
[fusion_filtering](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/fusion_filtering)
[gene-set-enrichment-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/gene-set-enrichment-analysis)
[gene_match](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/gene_match)
[immune-deconv](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/immune-deconv)
[independent-samples](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/independent-samples)

> [interaction-plots](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/interaction-plots)

Reviewer: ```suggestion
```

[molecular-subtyping-NBL](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-NBL)
[molecular-subtyping-chordoma](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-chordoma)
[molecular-subtyping-embryonal](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-embryonal)
[molecular-subtyping-integrate](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-integrate)
[molecular-subtyping-neurocytoma](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-neurocytoma)
[molecular-subtyping-pathology](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-pathology)
[mtp-annotations](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/mtp-annotations)
[mtp-tables-qc-checks](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/mtp-tables-qc-checks)
[mutational-signatures](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/mutational-signatures)

> [oncoprint-landscape](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/oncoprint-landscape)

Reviewer: ```suggestion
```

[pedcbio-cnv-prepare](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/pedcbio-cnv-prepare)
[pedcbio-sample-name](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/pedcbio-sample-name)
[pedot-table-column-display-order-name](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/pedot-table-column-display-order-name)
[rna-seq-expression-summary-stats](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/rna-seq-expression-summary-stats)
[rnaseq-batch-correct](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/rnaseq-batch-correct)
[run-gistic](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/run-gistic)
[snv-frequencies](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/snv-frequencies)

> [ssgsea-hallmark](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/ssgsea-hallmark)

Reviewer: ```suggestion
```

Author: Done.

> [survival-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/survival-analysis)

Reviewer: ```suggestion
```

Author: Done.

> [telomerase-activity-prediction](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/telomerase-activity-prediction)

Reviewer: ```suggestion
```

Author: Done.

[tmb-calculation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tmb-calculation)
[tp53_nf1_score](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tp53_nf1_score)
[tumor-gtex-plots](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tumor-gtex-plots)
[tumor-normal-differential-expression](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tumor-normal-differential-expression)

Software versions are documented in **Table XX**.

#### Data releases
We maintained a data release folder on Amazon S3, downloadable directly from S3 or our open-access CAVATICA project, with merged files for each analysis (See data and code availability section). 
As we produced new results that we expected to be used across multiple analyses, or identified data issues, we created new data releases in a versioned manner. 

> We reran all manuscript-specific analysis modules with the latest data release (v12) prior to submission and subsequently created a GitHub repository-tagged release to ensure reproducibility.

Reviewer: ```suggestion
```

Reviewer: this should be deleted for now

Author: Done. Please re-review when you can.

OpenPedCan analysis modules were developed within OpenPBTA [@doi:10.1016/j.xgen.2023.100340], modified based on OpenPBTA, or newly created and can be found within the following publicly available repositories.
OpenPBTA module analyses can be found at [https://github.com/AlexsLemonade/OpenPBTA-analysis](https://github.com/AlexsLemonade/OpenPBTA-analysis).
OpenPedCan module analyses can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis).
OpenPedCan api code can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-api](https://github.com/PediatricOpenTargets/OpenPedCan-api).

All original code was developed within the following modules in the OpenPedCan analyses repository as listed below. Links to the modules are available here, and within each module is a detailed README that describes the purpose and intended usage of the scripts, along with pointers to the results from the data those scripts process. 

> [chromosomal-instability](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/chromosomal-instability)

Reviewer: this is a good idea! can you remove the modules we did not use yet in OPC?

Author: @jharenza  I removed some of the modules, and now we are left with 52 modules that look OPC to me. Please review and confirm in case I missed removing some more or removed any OPC module.

[tcga-capture-kit-investigation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tcga-capture-kit-investigation)
[telomerase-activity-prediction](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/telomerase-activity-prediction)
[tmb-calculation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tmb-calculation)
[tmb-compare](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tmb-compare)
[tp53_nf1_score](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tp53_nf1_score)
[transcriptomic-dimension-reduction](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/transcriptomic-dimension-reduction)
[tumor-gtex-plots](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tumor-gtex-plots)
[tumor-normal-differential-expression](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tumor-normal-differential-expression)

Software versions are documented in **Table XX**.

#### Data releases

> We maintained a data release folder on Amazon S3, downloadable directly from S3 or our open-access CAVATICA project, with merged files for each analysis (See data and code availability section). As we produced new results that we expected to be used across multiple analyses, or identified data issues, we created new data releases in a versioned manner. We reran all manuscript-specific analysis modules with the latest data release (v12) prior to submission and subsequently created a GitHub repository-tagged release to ensure reproducibility.

Reviewer: ```suggestion
We maintained a data release folder on Amazon S3, downloadable directly from S3 or our open-access CAVATICA project, with merged files for each analysis (See data and code availability section). 
As we produced new results that we expected to be used across multiple analyses, or identified data issues, we created new data releases in a versioned manner. 
Following each data release, we created a repository release.
```
also breaking into one sentence per line
